Ooui.Wasm.Build.Tasks/BuildDistTask.cs

```diff
@@ -16,9 +16,15 @@ namespace Ooui.Wasm.Build.Tasks
 {
     public class BuildDistTask : Task
     {
-        const string SdkUrl = "https://xamjenkinsartifact.azureedge.net/test-mono-mainline-wasm/916/ubuntu-1804-amd64/sdks/wasm/mono-wasm-f25f9e5f2b5.zip";
+        const string SdkUrl = "https://jenkins.mono-project.com/job/test-mono-mainline-wasm/label=ubuntu-1804-amd64/lastSuccessfulBuild/Azure/processDownloadRequest/4942/ubuntu-1804-amd64/sdks/wasm/mono-wasm-eb468076a79.zip";
```

Comment:
This link (https://jenkins.mono-project.com/job/test-mono-mainline-wasm/label=ubuntu-1804-amd64/lastSuccessfulBuild/Azure/processDownloadRequest/4942/ubuntu-1804-amd64/sdks/wasm/mono-wasm-eb468076a79.zip) doesn't work anymore. The correct working link for the same Mono Wasm build is https://xamjenkinsartifact.blob.core.windows.net/test-mono-mainline-wasm/4942/ubuntu-1804-amd64/sdks/wasm/mono-wasm-eb468076a79.zip